src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -18,6 +18,7 @@
 from dsp_tools.commands.excel2xml.propertyelement import PropertyElement
 from dsp_tools.models.exceptions import BaseError
 from dsp_tools.models.helpers import DateTimeStamp
+from dsp_tools.utils.date_util import _full_date_pattern
```

Comment:
it should never be necessary to import a private member like this. Either we make the pattern public, or - which I would prefer - we expose a function that does the check for you. Then the caller also doesn't have to know that they need the verbose flag for the regex

Reply:
I agree.

Reply:
Thanks for the suggestion, I adapted it.